extensions/github1s/src/helpers/fetch.ts

```diff
@@ -108,6 +108,10 @@ export const fetch = reuseable(async (url: string, options?: RequestInit) => {
 	}
 	if (response.status === 401) {
 		return response.json().then((data) => {
+			// current token is invalid
+			if (data.message === 'Bad credentials') {
```

Comment:
Should we use `data.message?.includes('Bad credentials')` instead of `===` since the message might change.

Reply:
yeah, it is the better way to write it. We might face some issues with `===` !